client/src/components/Q&A/QA.jsx

```diff
@@ -2,17 +2,19 @@ import React, { useState } from 'react';
 import SearchBar from './SearchBar';
 import QuestionContainer from './QuestionContainer';
 import AddQModal from './AddQModal';
+import AddAnswerModal from './AddAnswerModal';
 
-export default function QA({ show, setShow }) {
+export default function QA() {
   const [searchTerm, setSearchTerm] = useState('');
   return (
     <div
       className="main-container"
     >
       <h2>Questions & Answers</h2>
```

Comment:
same here, mush 10 - 12 onto the same line, since it won't be too long and there is only one thing being added to the tag.

Reply:
not sure why I did that... lol